wp-content/plugins/core/src/Templates/Components/Abstract_Controller.php

```diff
@@ -59,4 +71,96 @@ public static function factory( array $args = [] ) {
 		return tribe_project()->container()->make( static::class, [ 'args' => $args ] );
 	}
 
+	public function render_breadcrumbs(): void {
```

Comment:
This is more of an opinion, but I think we shouldn't load templates within the controller and just add this call to the template.